frameworks/cassandra/universe/config.json

```diff
@@ -55,6 +55,12 @@
           "description": "Enable DC/OS Virtual Networking",
           "type": "boolean",
           "default": false
+        },
+        "log_level": {
+          "description": "The log level for the DC/OS service.",
```

Comment:
This name and description imply that it'll carry through to the underlying service. Maybe go with `scheduler_log_level`/`The log level for the service scheduler` or similar